### DISCUSSION TITLE
How to uninstall .deb applications?

### DISCUSSION THREAD

**Discussion by AlexValue** [Vanilla-OS/live-iso#365] [Q&A] [answered]
I installed a .deb app from the developer's website (https://lantern.io/), but now I want to uninstall it. In Fedora (with .rpm) I did this by right-clicking on the app, then the "more about the app" tab and the app center opened, where there was a "uninstall" button.

In VanillaOS, the app center doesn't seem to work with .deb apps. Is there a way to uninstall the app via the GUI?

**Comment by GabsEdits** [comment, marked as answer]
There isn't sadly any way yet to uninstall .deb applications via the GUI. You can remove them from the terminal using `apt remove <app>` or `vso remove <app>`
